### ISSUE TITLE
Widescreen UI problem in MB2 with OpenJK client

### ISSUE THREAD

**Issue by wrrersrrer** [JACoders/OpenJK#891]
In Movie Battles 2 there is a Widescreen UI option which changes the width of the UI so it doesn't look stretched in widescreen resolutions. That option works in the JAMP version of the mod's client, but it doesn't work in the OpenJK version of the client.
Here (https://community.moviebattles.org/threads/openjk-vs-jamp.910/#post-9154) I found the explanation of this issue:
"- it [OpenJK] is lacking of some features like ratio fix support (though they started adding it recently, but not completed), so everything looks stretched in the game"
Are you guys going to complete the ratio fix support system? I don't understand why you removed this system from OpenJK if it had existed in JAMP.
Here are all the threads on MB2 forums regarding this issue:
https://community.moviebattles.org/threads/widescreen-ui-on-my-resolution.3283/
https://community.moviebattles.org/threads/hud-is-still-stretched-out-and-disproportionate.860/
https://community.moviebattles.org/threads/difference-between-jamp-client-and-mbii-client.1231/
https://community.moviebattles.org/threads/openjk-vs-jamp.910/
This issue is also probably related to this issue:
https://github.com/JACoders/OpenJK/issues/36

**Comment by ensiform** [comment]
This system doesn't exist in JAMP.  They utilize engine patching to make it work for JAMP, but not OpenJK.

**Comment by wrrersrrer** [comment]
So, you are saying it's on their side and not yours?

**Comment by ensiform** [comment]
Yes, it has absolutely nothing to do with JAMP or OpenJK.  They fixed it in MB2 to specifically fix it in JAMP.

**Comment by wrrersrrer** [comment]
Well, I don't know how it came to be thought like that, it seems some people got it wrong from the context, or something, and I did too. It seems that that post I linked above (https://community.moviebattles.org/threads/openjk-vs-jamp.910/#post-9154) stating the pros and cons of OpenJK was misinterpreted or something, since it says that "- it [OpenJK] is lacking of some features like ratio fix support (though they started adding it recently, but not completed), so everything looks stretched in the game" as the con of OpenJK, and the thread itself is about the differences between OpenJK and JAMP, so people thought that JAMP has it, or that the author of that post thought that JAMP has that system since MB2's Widescreen UI option only works in JAMP. I can only guess, though, since it can be interpreted pretty ambiguously.
